online-members/online-members.css

```diff
@@ -123,3 +123,31 @@ footer .info-repo {
   font-weight: 100;
   text-align: center;
 }
+
+/* MEDIA QUERY */
+
+@media only screen and (max-width: 900px) {
+  .body {
+    flex-direction: column-reverse;
+    align-items: center;
+    padding-top: 2rem;
+    gap: 3rem;
```

Comment:
why gap of 3rem is needed... any specific reason?? why not 10px or 5px??

Reply:
I have given spacing between two containers which are in column in mobile view. Spacing of 10px is not enough.